File: src/VSServiceStack/ProjectTemplates/FSharpEmptyAspNet/FSharpEmptyAsp/AppHost.fs
namespace $safeprojectname$
    open ServiceStack
    open ServiceStack.Common
    open $safeprojectname$
    open $safeprojectname$.ServiceInterface
    open System

    type AppHost() = 
        inherit AppHostBase("$safeprojectname$", typeof<MyServices>.Assembly)
        override this.Configure container =
            //Add plugins
            //this.Plugins.Add(new PostmanFeature())
            //this.Plugins.Add(new CorsFeature())
            ignore()
        static member Start() = 
            let appHost = new AppHost()
            appHost.Init() |> ignore

File: src/ServiceStackVS/ProjectTemplates/AspNetMVC4/Host/App_Start/AppHost.cs
using System.Configuration;
using System.Web.Mvc;
using Funq;
using ServiceStack;
using ServiceStack.Auth;
using ServiceStack.Configuration;
using ServiceStack.Mvc;
using $safeprojectname$.ServiceInterface;
using $safeprojectname$.ServiceModel;

[assembly: WebActivator.PreApplicationStartMethod(typeof($safeprojectname$.AppHost), "Start")]
//More info on how to integrate with MVC: https://github.com/ServiceStack/ServiceStack/wiki/Mvc-integration

namespace $safeprojectname$
{
    public class AppHost : AppHostBase
    {		
        /// <summary>
        /// Base constructor requires a Name and Assembly where web service implementation is located
        /// </summary>
        public AppHost()
            : base("$safeprojectname$", typeof(MyServices).Assembly) {}

        /// <summary>
        /// Application specific configuration
        /// This method should initialize any IoC resources utilized by your web service classes.
        /// </summary>
        public override void Configure(Container container)
        {
            SetConfig(new HostConfig
            {
                HandlerFactoryPath = "api",
            });
            //Config examples
            //this.Plugins.Add(new PostmanFeature());
            //this.Plugins.Add(new CorsFeature());

            //Set MVC to use the same Funq IOC as ServiceStack
            ControllerBuilder.Current.SetControllerFactory(new FunqControllerFactory(container));
        }

        public static void Start()
        {
	        new AppHost().Init();
        }
    }
}
